### PR TITLE
Remove unused key to avoid duplicate keys warning

### PR DESCRIPTION
**Pull request by kesselb**
image

```

[Vue warn]: Duplicate keys detected: 'some@mail.com'. This may cause an update error.

found in

---> <EventData> at src/components/imip/EventData.vue
       <Imip> at src/components/Imip.vue
         <Message> at src/components/Message.vue
```

### CODE REVIEW

**Comment by kesselb** [comment]
> Elements in iteration expect to have 'v-bind:key' directives  

:angry:

**Comment by kesselb** [comment]
Should be good now :partying_face:

**Review by ChristophWurst — APPROVED** [review]

File: src/components/imip/EventData.vue [hunk 1]
```diff
@@ -49,7 +49,7 @@
 		<div class="event-data__row event-data__row--participants">
 			<AccountMultipleIcon class="event-data__row__icon" :size="20" />
 			<ul>
-				<li v-for="{ name, isOrganizer } in attendees" :key="name">
+				<li v-for="{ name, isOrganizer } in attendees" :key="key">
```
Comment (ChristophWurst) [inline src/components/imip/EventData.vue:52]:
```suggestion
				<li v-for="{ name, isOrganizer, key } in attendees" :key="key">
```

Comment (kesselb) [reply src/components/imip/EventData.vue]: Thanks!

**Pull request merged**